transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/RestTransportClient.java

```diff
@@ -77,7 +77,7 @@ private static HttpClientOptions createHttpClientOptions() {
     HttpClientOptions httpClientOptions = new HttpClientOptions();
     httpClientOptions.setMaxPoolSize(TransportClientConfig.getConnectionMaxPoolSize())
         .setIdleTimeout(TransportClientConfig.getConnectionIdleTimeoutInSeconds())
-        .setKeepAliveTimeout(TransportClientConfig.getConnectionIdleTimeoutInSeconds())
+        .setKeepAliveTimeout(TransportClientConfig.getHttp1ConnectionKeepAliveTimeoutInSeconds())
```

Comment:
This change should create PR to master/2.8.x/1.3.x

Reply:
ok